elixir/lib/my_app/router.ex

```diff
@@ -0,0 +1,12 @@
+defmodule MyApp.Router do
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(%Plug.Conn{request_path: "/_ah/health"} = conn, _opts) do
+    send_resp(conn, 200, "👌")
```

Comment:
Return the string "ok", as I think earlier versions of App Engine expected it. I don't think it actually matters anymore, but better to be consistent than clever in an example.